janusgraph-benchmark/src/main/java/org/janusgraph/BenchmarkRunner.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph;
+
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+public class BenchmarkRunner {
+    public static void main(String[] args) throws RunnerException {
+        String firstArg = ".*Benchmark";
+        if (args.length > 0) {
+            firstArg = args[0];
+        }
+        Options opt = new OptionsBuilder()
+            .include(firstArg)
```

Comment:
This looks fine, but I have a nitpick: if we could use a for loop to chain the `include` statements rather than using `firstArg` only, then we can run BenchmarkRunner with an arbitrary number of benchmark classes.

Reply:
done